src/components/item/IFXItemListMixin.js

```diff
@@ -74,7 +74,7 @@ export default {
       return formattedItems
     },
     navigateToItemCreate() {
-      this.rtr.push({ name: `${this.itemType}Create` })
+      this.rtr.push({ name: `${this.itemType}Create`, query: { next: this.$route.path } })
```

Comment:
I'm doing similar things in the branch I have for button changes.

Reply:
Will sort out in the merge